### ISSUE TITLE
Token address replacement question

### ISSUE THREAD

**Issue by bbgdex** [TP-Lab/tokens#7905]
Can the address of the token that has been uploaded be changed? If so, will it still cost?

**Comment by opera0001** [comment]
If you submit a contract address that has already been merged, a new donation will be required to amend it again. If you are submitting a logo that needs to be modified, you can contact me within 24 hours after it has been merged to modify or optimize it.

If the address you provided is not merged in the order, then you can modify it yourself to replace the incorrect contract address.

**Comment by bbgdex** [comment]
> If you submit a contract address that has already been merged, a new donation will be required to amend it again. If you are submitting a logo that needs to be modified, you can contact me within 24 hours after it has been merged to modify or optimize it.
> 
> If the address you provided is not merged in the order, then you can modify it yourself to replace the incorrect contract address.

The token addresses have been merged

Do we still need to donate TPT 50,000.00 again?

**Comment by opera0001** [comment]
Yes, you still need to submit a new tpt hash if you need to change the contract address.

**Comment by bbgdex** [comment]
> Yes, you still need to submit a new tpt hash if you need to change the contract address.

Will the original contract address be deleted ？

**Comment by opera0001** [comment]
> > Yes, you still need to submit a new tpt hash if you need to change the contract address.
> 
> Will the original contract address be deleted ？

Generally the logo is removed from the contract, as some users still hold these tokens, so the contract address is generally not removed.

**Comment by bbgdex** [comment]
> > > Yes, you still need to submit a new tpt hash if you need to change the contract address.
> > 
> > 
> > Will the original contract address be deleted ？
> 
> Generally the logo is removed from the contract, as some users still hold these tokens, so the contract address is generally not removed.

I got it.
